gr-digital/lib/crc_check_impl.h

```diff
@@ -44,7 +44,7 @@ class crc_check_impl : public crc_check
                      gr_vector_int& ninput_items,
                      gr_vector_const_void_star& input_items,
                      gr_vector_void_star& output_items);
-    void msg_handler(pmt::pmt_t pmt_msg);
+    void msg_handler(const pmt::pmt_t& pmt_msg);
```

Comment:
API backward compatible?

Reply:
nope, hence singled-out in discardable commit.

Reply:
Is the `const` required for the `ok_port`/`fail_port` initializaiton, or just a good idea?

Reply:
ah, no, it's not required, but I didn't want to make two miniature performance-enhancing commits, can split if you actually want to backport only the static initialization enhancement.